app/src/main/res/values/strings.xml

```diff
@@ -84,4 +84,10 @@
     <string name="digital_outputs">Digital Outputs</string>
     <string name="configure_PWM">Configure PWM</string>
     <string name="waveform_generator">Arbitrary Waveform Generator</string>
+    <string name="autoscan">AUTOSCAN</string>
+    <string name="select_sensor">SELECT SENSOR</string>
+    <string name="get_raw">GET RAW</string>
```

Comment:
We can undo these strings to be in normal case and set attributes in buttons to view them as uppercase! What do you think? :)